alacritty_terminal/src/term/mod.rs

```diff
@@ -2264,6 +2280,41 @@ impl TabStops {
             is_tabstop
         });
     }
+
+    fn tab_range(&self, grid: &Grid<Cell>, point: Point) -> Option<IndexRange<Column>> {
```

Comment:
I was not sure where to put this function, it feels wrong here, but also not completely out of place.

Also not sure if it should take `Point<usize>` instead

Reply:
Is the point of this function to find the start and end of the tab the `point` is in? This seems a bit complicated.

Also isn't it pointless to get the entire range of the tab, when the only goal is to determine if a point lies within a selected tab?

Reply:
But you need to first figure out if the tab is selected.

Correct me if I am wrong here:

Every cell gets checked if it is selected, so in order to know if the current cell is selected, we have to check if the cell is part of a tab (go to the left and find a \t) and if the current cell is a tab, we have to find out if any cell of the tab is already selected.

E.g.:
* x = current cell
* y = selected
* \t = tab begin

`\t.....x......y` 

* part of a tab: to the right of a \t without a tab border
* any cell of the tab is selected (y)

Now if there is a higher level way of marking a tab and all of its cells as selected, this would obviously be easier. I've seen there are modifiers as used for wide chars, but it didnt seem like they are meant for this usecase.